CONTRIBUTING.md

```diff
@@ -121,6 +123,6 @@ Name your branches with prefixes: `[type]/[area]/[description]`
 
 * `type` = feature, bug, refactoring, benchmark, security, documentation, dependencies, ci, ...
 * `area` (\*) = gateway, meta, manager-backup, manager-replication, ...
-* `description` = branch description. description must be hyphenated.
+* `description` = branch description. description must be hyphenated. please use [a-zA-Z0-9] and hyphen as characters, do not use any other characters.
```

Comment:
```suggestion
* `description` = branch description. The description must be hyphenated. please use [a-zA-Z0-9] and hyphen as characters, do not use any other characters.
```

Reply:
```suggestion
* `description` = branch description. The description must be hyphenated. Please use [a-zA-Z0-9] and hyphen as characters, do not use any other characters.
```

Reply:
@hlts2 How is it?

Reply:
LGTM, but I have a idea.
IMO: using table style would be more user friendly

```html
<table>
<tr><td>type<td>type represents xxx<td>feature, bug, refactoring, benchmark, security, documentation, dependencies, ci, (XXX define every patter)
<tr><td>area (*)<td>this field represents area or context<td>gateway, meta, manager-backup, manager-replication, ... (XXX show more example)
<tr><td>description<td>summarized description of branch<td> branch description. description must be hyphenated. please use [a-zA-Z0-9] and hyphen as characters, do not use any other characters.
</table>
```

Reply:
sorry but I want to confirm `description` means the branch name?

Reply:
> How is it?

it seems good to me!
and I also think that the table structure was easier to understand!

Reply:
@kevindiu 
It is almost right.
branch name is [type]/[area]/[description]